src/main/resources/com/google/api/codegen/js/sample.snip

```diff
@@ -114,16 +114,37 @@
 
       @end
       function {@class.initClientFuncName}() {
+        @if class.auth.type != "NONE"
+          var API_KEY = '';  // TODO: Update placeholder with desired API key.
```

Comment:
Okay --- I still don't understand, but it seems to have been thoroughly reviewed.